vortex-ffi/src/file.rs

```diff
@@ -35,6 +35,7 @@ pub struct vx_file_reader {
 
 /// Options supplied for opening a file.
 #[repr(C)]
```

Comment:
I bet this implies non camel case

Reply:
Let me double check whether this isn't an editor issue..

Reply:
Which is though bc of the enum values not the type.